db/migrate/20150701155145_add_last_url_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddLastUrlToUsers < ActiveRecord::Migration
+  tag :predeploy
+  def change
+    add_column :users, :last_url, :string
```

Comment:
I'm just going to assume that you looked into the object model and that users is the best object for last_url and last_url_title rather than just the first, easiest option.